Formula/c/chainsaw.rb

```diff
@@ -19,6 +19,11 @@ class Chainsaw < Formula
   depends_on "rust" => :build
 
   def install
+    # Upstream forgot to patch the version parameter in the Cargo.toml file
```

Comment:
can we log an issue for this?

Reply:
Open an issue, added it to comment, shorter it.